Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ros:dashing
+
+USER root
+RUN apt update -qq && apt-get install -y -q \
+    build-essential \
+    cmake \
+    git \
+    apt-transport-https \
+    software-properties-common \
+    locales \
+    openssh-client \
+    python3-pip \
+    wget \
+    ros-dashing-turtlebot3* \
+    ros-dashing-gazebo-ros* \
+    ros-dashing-rviz2 \
+    && pip3 install -U pip setuptools \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN locale-gen en_US.UTF-8; dpkg-reconfigure -f noninteractive locales
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
+RUN groupadd -g 1000 rbccps && \
+    useradd -d /home/rbccps -s /bin/bash -m rbccps -u 1000 -g 1000 && \
+    usermod -aG sudo rbccps && \
+    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+USER rbccps
+RUN mkdir -p /home/rbccps/mrs_ws/src
+ENV HOME /home/rbccps
+WORKDIR /home/rbccps/mrs_ws/
```

Comment:
1) Is it necessary to perform all these steps (25 to 33) to create the user and the home?
2) Please change the name of the user to "choirbot" and the name of the workspace to "dev_ws"